stablehlo/dialect/StablehloOps.td

```diff
@@ -323,7 +323,8 @@ def StableHLO_ExpOp: StableHLO_UnaryElementwiseOp<"exponential",
 }
 
 def StableHLO_Expm1Op: StableHLO_UnaryElementwiseOp<"exponential_minus_one",
-    [Pure, HLO_CompatibleOperandsAndResultType], HLO_FpOrComplexTensor> {
+    [Pure, HLO_CompatibleOperandsAndResultType],  /*exponential_minus_one_c1*/
```

Comment:
```
    [Pure, HLO_CompatibleOperandsAndResultType], /*exponential_minus_one_c1*/
	HLO_FpOrComplexTensor /*exponential_minus_one_i1*/> { /*exponential_minus_one_c1*/
```

Feedback:
* typo (double space)
* `StableHLO_UnaryElementwiseOp` class also has the `SameOperandsAndResultShape` which addresses (C1). Added comment following the [checklist](https://github.com/openxla/stablehlo/blob/main/docs/reference_checklist.md).